pages/api/stats.ts

```diff
@@ -1,34 +1,10 @@
 import { withApiMethods } from "../../lib/withApiMethods";
-import { prisma } from "../../lib/db";
-import { generateStats } from "../../lib/stats/stats";
+import * as stats from "../../lib/stats/stats";
```

Comment:
I don't like this `import * as stats from "..."` approach. We're just using the update function here anyways... And that function's name isn't precise either, it doesn't just update the stats. What it actually does, is it **gets** _or_ **updates and returns** the stats.

This level of separation from the route controller isn't intuitive IMO. The better approach would be:
- Controller gets the stats from DB
- Check if the stats are expired - if so - use the generate function and send its results
- If the stats aren't expired - send them